File: src/test/java/com/github/krystianmuchla/home/IntegrationTest.java
package com.github.krystianmuchla.home;

import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;
import org.testcontainers.junit.jupiter.Container;
import org.testcontainers.junit.jupiter.Testcontainers;

@SpringBootTest
@Testcontainers
@ActiveProfiles("test")
public class IntegrationTest {
    @Container
    private static final DatabaseContainer databaseContainer = new DatabaseContainer();
}

File: src/test/kotlin/com/example/skyr/IntegrationTest.kt
package com.example.skyr

import org.springframework.boot.test.context.SpringBootTest
import org.testcontainers.junit.jupiter.Container
import org.testcontainers.junit.jupiter.Testcontainers

@SpringBootTest
@Testcontainers
class IntegrationTest {

    companion object {

        @Container
        private val databaseContainer = DatabaseContainer()
    }
}
